client/session.go

```diff
@@ -66,9 +66,21 @@ func (x ResSessionCreate) PublicKey() []byte {
 // see [apistatus] package for NeoFS-specific error types.
 //
 // Context is required and must not be nil. It is used for network communication.
+//
+// Return errors:
+//   - [ErrMissingSigner]
 func (c *Client) SessionCreate(ctx context.Context, prm PrmSessionCreate) (*ResSessionCreate, error) {
+	signer := prm.signer
+	if signer == nil {
+		signer = c.prm.signer
+	}
+
+	if signer == nil {
```

Comment:
Maybe we can make `func (c *Client) getSigner(prm neofscrypto.Signer) (neofscrypto.Signer, error)` to be used like
```
signer, err := c.getSigner(prm.signer)
if err != nil {
        return nil, err
}
```